packages/oas-to-snippet/src/index.ts

```diff
@@ -78,13 +78,17 @@ export default function oasToSnippet(
     );
   }
 
+  if (!language || !target) {
+    return { code: '', highlightMode: false };
+  }
+
   const har = opts.harOverride || generateHar(oas, operation, values, auth);
   const snippet = new HTTPSnippet(har as HarRequest, {
     // We should only expect HAR's generated with `@readme/oas-to-har` to already be encoded.
     harIsAlreadyEncoded: !opts.harOverride,
   });
 
-  let targetOpts = config.httpsnippet.targets[target].opts || {};
+  let targetOpts = config.httpsnippet.targets[target || ''].opts || {};
```

Comment:
`target` shouldn't be undefined here anymore with the logic you added above.

Reply:
good catch, will revert

Reply:
https://github.com/readmeio/oas/pull/857/commits/9ee31d3473a00219918201e1c17cfc3ff46ee325